### ISSUE TITLE
Electrum 3.0.5 not opening on Mac OS 10.9

### ISSUE THREAD

**Issue by lorepieri8** [spesmilo/electrum#3685]
As in the title, Electrum is not opening in Mac OS 10.9.

I had a similar issue with Electrum LTC, that was solved with an additional build (3.5.0.1-1, not available from the main page):

[https://github.com/pooler/electrum-ltc/issues/125](url)

Hopefully the solution for BTC should be similar.

**Comment by lorepieri8** [comment]
It may be related to #3673

**Comment by lzsaver** [comment]
So does it related to https://github.com/pooler/electrum-ltc/issues/125?

**Comment by lorepieri8** [comment]
It seems so. In the console I get the same error codes:

16/01/18 19:32:57,176 gkbisd[144]: Unable to collect cdhash for /Applications/Electrum.app (error code 100024)

16/01/18 19:32:57,315 com.apple.launchd.peruser.501[157]: (Electrum.86976[62489]) Exited with code: 255

**Comment by tessus** [comment]
I've just tried Electrum 3.1.1 on MacOSX 10.9.5, but I only get the following message (nothing from gkbisd):

```
Exited with code:255
```

How can I debug this? Or does anybody know what the problem might be?

**Comment by tessus** [comment]
Is there any update on this? Maybe @ecdsa can create a build that works on 10.9.5.

I tried to create one on my 10.9.5 machine, but it didn't work. After a few hours I gave up.

**Comment by bauerj** [comment]
Yeah, we should add that to the download page but currently on El Capitan (10.11.6) or newer is supported. To build a version that is compatible to older versions, we would need an installation of that version, which we do not have.

May I ask why you need to use Mavericks?

**Comment by tessus** [comment]
Long story short, I still use a MacBook (Late 2008) and I'm afraid to upgrade that from 10.9.5.

(I also have another MacBook that runs 10.13.4)

**Comment by SomberNight** [comment]
Note: need to keep an eye on this when increasing the PyQt version
See https://github.com/spyder-ide/qtpy/issues/155

**Comment by SomberNight** [comment]
For future reference:

Qt 5.8 needs [minimum MacOS 10.9](https://doc.qt.io/archives/qt-5.8/supported-platforms.html)
Qt 5.9 needs [minimum MacOS 10.10](https://doc.qt.io/archives/qt-5.9/supported-platforms.html)
Qt 5.10 needs [minimum MacOS 10.11](https://doc.qt.io/archives/qt-5.10/supported-platforms.html)
Qt 5.11 needs [minimum MacOS 10.11](https://doc.qt.io/archives/qt-5.11/supported-platforms.html)
Qt 5.12 needs [minimum MacOS 10.12](https://doc.qt.io/qt-5.12/supported-platforms.html)
Qt 5.13 needs [minimum MacOS 10.12](https://doc.qt.io/archives/qt-5.13/supported-platforms.html)
Qt 5.14 needs [minimum MacOS 10.13](https://doc.qt.io/archives/qt-5.14/supported-platforms.html)
Qt 5.15 needs [minimum MacOS 10.13](https://doc.qt.io/qt-5.15/supported-platforms.html)
Qt 6.0 needs [minimum MacOS 10.14](https://doc.qt.io/archives/qt-6.0/supported-platforms.html)
Qt 6.1 needs [minimum MacOS 10.14](https://doc.qt.io/archives/qt-6.1/supported-platforms.html)
Qt 6.2 needs [minimum MacOS 10.14](https://doc.qt.io/qt-6.2/supported-platforms.html)
Qt 6.3 needs [minimum MacOS 10.14](https://doc.qt.io/qt-6.3/supported-platforms.html)
Qt 6.4 needs [minimum MacOS 10.14](https://doc.qt.io/qt-6.4/supported-platforms.html)
Qt 6.5 needs [minimum MacOS 11](https://doc.qt.io/qt-6.5/supported-platforms.html)
Qt 6.6 needs [minimum MacOS 11](https://doc.qt.io/qt-6.6/supported-platforms.html)
Qt 6.7 needs [minimum MacOS 11](https://doc.qt.io/qt-6.7/supported-platforms.html)
Qt 6.8 needs [minimum MacOS 12](https://doc.qt.io/qt-6.8/supported-platforms.html)
Qt 6.9 needs [minimum MacOS 12](https://doc.qt.io/qt-6.9/supported-platforms.html)
Qt 6.10 needs [minimum MacOS 13](https://doc.qt.io/qt-6/supported-platforms.html)
